app/diagnostics.go

```diff
@@ -251,12 +251,15 @@ func trackConfig() {
 	})
 
 	SendDiagnostic(TRACK_CONFIG_FILE, map[string]interface{}{
-		"enable_public_links":     utils.Cfg.FileSettings.EnablePublicLink,
-		"driver_name":             utils.Cfg.FileSettings.DriverName,
-		"amazon_s3_ssl":           *utils.Cfg.FileSettings.AmazonS3SSL,
-		"amazon_s3_signv2":        *utils.Cfg.FileSettings.AmazonS3SignV2,
-		"max_file_size":           *utils.Cfg.FileSettings.MaxFileSize,
-		"enable_file_attachments": *utils.Cfg.FileSettings.EnableFileAttachments,
+<<<<<<< HEAD
+		"enable_public_links": 		 utils.Cfg.FileSettings.EnablePublicLink,
+		"driver_name":         		 utils.Cfg.FileSettings.DriverName,
+		"amazon_s3_iam_profile": 	 *utils.Cfg.Filesettings.AmazonS3IamProfile,
+		"amazon_s3_ssl":       		 *utils.Cfg.FileSettings.AmazonS3SSL,
+		"amazon_s3_signv2":    		 *utils.Cfg.FileSettings.AmazonS3SignV2,
+		"max_file_size":       		 *utils.Cfg.FileSettings.MaxFileSize,
+		"enable_file_attachments": *utils.Cfg.FileSettings.EnableFileAttachments
```

Comment:
Also, there's a build failure right now due to a failed import, but this will cause Jenkins to reject the build since it fails `gofmt`